pkg/reconcilermanager/controllers/reposync_controller.go

```diff
@@ -651,6 +661,14 @@ func (r *RepoSyncReconciler) mapSecretToRepoSyncs(secret client.Object) []reconc
 	return requests
 }
 
+func (r *RepoSyncReconciler) mapAdmissionWebhookToRepoSyncs(admissionWebhook client.Object) []reconcile.Request {
+	sRef := client.ObjectKeyFromObject(admissionWebhook)
```

Comment:
You can use `admissionWebhook.GetName()` to replace `sRef.Name`.

Reply:
Done